packages/cli/test/utils/crucible/kurtosis/runner/kurtosisSDKRunner.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ * KurtosisSDKRunner - Ethereum Network Simulation Runner
+ * =====================================================
+ *
+ * An implementation of the KurtosisSDKRunner that leverages
+ * Kurtosis SDK for creating and managing Ethereum network simulations.
+ * It acts as a replacement for the current Runner.
+ *
+ * This runner provides:
+ * - Enclave lifecycle management (create, start, stop, destroy)
+ * - Service deployment using ethpandaops/ethereum-package
+ * - Automatic service role inference and metadata extraction
+ * - Resource cleanup and error handling
+ *
+ */
+
+import {EnclaveContext, KurtosisContext, StarlarkRunConfig} from "kurtosis-sdk";
+import {IRunner, RunnerEvent} from "../simulation/interfaces-kurtosis.js";
+import {KurtosisNetworkConfig, KurtosisServicesMap, NodeService} from "./kurtosisTypes.js";
+
+export class KurtosisSDKRunner implements IRunner {
+  private enclaveName: string;
+  private kurtosisContext?: KurtosisContext;
+  private enclaveContext?: EnclaveContext;
+
+  /**
+   * Creates a new KurtosisSDKRunner instance.
+   *
+   * enclaveName: Default name for the Kurtosis enclave - can be overridden
+   */
+  constructor(enclaveName = "crucible-enclave") {
```

Comment:
Make the name required, so the enclave name already be unique with the simulation running. And don't mixup the services among different simulations.